src/bud/state/env.ts

```diff
@@ -0,0 +1,18 @@
+import {join} from 'path'
+import dotenv from 'dotenv'
+import {paths} from './paths'
+
+/**
+ * Environment variables container.
+ */
+const envRaw = dotenv.config({
+  path: join(paths.project, '.env'),
+}).parsed
+
+const env: Environment = {
+  ...envRaw,
+}
+
+export {env}
```

Comment:
I assume that `envRaw` and `env` have been separated so that additional properties might be added to `env` in the future? It might be helpful to explain your intention in a comment.

Reply:
This is actually just an artifact of me trying to type `env` with `Environment` (L18), but I think I could just assign the parsed values to `env` and typecast during assignment. This is superfluous.